users/service.go

```diff
@@ -284,11 +298,11 @@ func (svc service) ResetSecret(ctx context.Context, resetToken, secret string) e
 		return errors.Wrap(svcerr.ErrViewEntity, err)
 	}
 	if c.Credentials.Identity == "" {
-		return repoerr.ErrNotFound
+		return svcerr.ErrNotFound
```

Comment:
Remove this check no need to check `c.Credentials.Identity` is empty as it is always present